apps/pie-docs/src/components/icon-button/overview/overview.md

```diff
@@ -0,0 +1,357 @@
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Hover</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/interactive-state-hover.svg",
+      width: 48,
+      alt: "An icon button that is hovered."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Active</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/interactive-state-active.svg",
+      width: 48,
+      alt: "An icon button that is active."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Focus</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/interactive-state-focus.svg",
+      width: 72,
+      alt: "An icon button that is focused."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Disabled</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/interactive-state-disabled.svg",
+      width: 48,
+      alt: "An icon button that is disabled."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Loading</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/interactive-state-loading.svg",
+      width: 48,
+      alt: "An icon button that is loading."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Example
+
+Outlines the atomic level interactive elements for the component.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/example-hamburger-menu.svg",
+      width: 48,
+      alt: "A hamburger menu icon button."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/example-close.svg",
+      width: 48,
+      alt: "A close icon button."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/icon-button/example-icon-button-pair.svg",
+      width: 104,
+      alt: "Info and heart icon buttons."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+--- 
+
+## Resources
+
+{% resourceTable {
+    rows: [
+        {
+            resource: resourceTypes.COMPONENT,
+            link: "https://www.figma.com/file/R2rBfzJP0hG0MZorq6FLZ1/%5BCore%5D-Components-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=981-5350&mode=design",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.DOCUMENTATION,
+            link: "https://www.figma.com/file/j1YKygEyhqZ6zKVxcHapn5/%5BCore%5D-Component-Documentation-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=742-317&mode=design",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.WEB_COMPONENTS,
+            link: "https://webc.pie.design/?path=/story/icon-button--primary",
+            status: statusTypes.ALPHA
```

Comment:
In the designs it looks like this page only has 5 links (Figma x2, Webc, Fozzie and Snacks) and they are all `AVAILABLE`

Reply:
yes but the web component status is still in ALPHA so we can't move to available.